### ISSUE TITLE
Rename Angular 2 HTML component files

### ISSUE THREAD

**Issue by agoncal** [jhipster/generator-jhipster#4499]
##### **Overview of the issue**

ATM with the Angular branch, component files are named :

```
├── activate
│   ├── activate.component.ts
│   ├── activate.html
├── password
│   ├── password.component.ts
│   ├── password.html
```

Following the Angular CLI standards, the HTML files should be called the name of the component with an `html` suffix : 

```
├── activate
│   ├── activate.component.ts
│   ├── activate.component.html
├── password
│   ├── password.component.ts
│   ├── password.component.html
```

##### **Motivation for or Use Case**

When using the Angular CLI with the following commands : 

```
ng g component activate
ng g component password
```

We get the following : 

```
├── activate
│   ├── activate.component.ts
│   ├── activate.component.html
├── password
│   ├── password.component.ts
│   ├── password.component.html
```

Sticking to Angular CLI would be better

**Comment by deepu105** [comment]
Yes it needs to be improved. PRs welcome :)

Thanks & regards,
Deepu

**Comment by deepu105** [comment]
closed via #4503
